source/deployment/resyncing-conda.rst

```diff
@@ -8,29 +8,35 @@ operating cycle, to help isolate users from changes that might break things
 mid-cycle due to some unintended upgrade.
 
 Before the start of each cycle, we re-synchronize our internal collection of
-packages with the latest packages available from Anaconda.
+packages with the latest packages available from Anaconda (defaults channel).
 
 The internal Anaconda server runs on the ``alexandria`` host, inside the Controls
-network, and it runs as a service account user ``anaconda-server``.
+network, and it runs as root.
 
 .. code-block:: bash
 
     ssh alexandria
-    sudo su anaconda-server
-    cd
-    export https_proxy=http://proxy:8888
-    export HTTPS_PROXY=http://proxy:8888
+    sudo su
+    bash /opt/builds/bootstrap-mirror-nsls2anaconda-local-folder
+    bash /opt/builds/run_conda_index
 
-The configuration file ``~/anaconda.yaml`` controls which packages are synced.
-Edit it, for example, to update which version(s) of Python we want packages for
-this cycle.
+bootstrap-mirror-nsls2anaconda-local-folder script can be easily written
+based on https://github.com/NSLS-II/lightsource2-recipes/blob/master/scripts/bootstrapping/bootstrap-tag-build and
```

Comment:
Sphinx has some functionality for the links, maybe it can be used for long links like this one?
See an example here: https://github.com/NSLS-II/docs/blob/85093efe8c5e7133a6af9a507f9bb82f7ecd1d7b/source/_cookbook/live_recon.py#L11